src/CareerBuilder/OAuth2/Flows/Flow.php

```diff
@@ -118,10 +144,13 @@ public function getToken(AccessToken $token = null)
             $this->buildBody();
         }
 
-        $request = $this->client->post($this->tokenRequestPath, $this->headers, $this->body);
-        $response = $request->send();
-        $data = $response->json();
-
+        /** @var Response $response */
+        $response = $this->client->post($this->tokenRequestPath,[
```

Comment:
Is this G6 compatible??

Reply:
Yeah, tokenRequestPath is a string, and the parameters was named correctly